mysql/resultset_helper.go

```diff
@@ -39,13 +42,53 @@ func FormatTextValue(value interface{}) ([]byte, error) {
 		return v, nil
 	case string:
 		return utils.StringToByteSlice(v), nil
+	case time.Time:
+		return hack.Slice(v.Format(time.DateTime)), nil
```

Comment:
hack is deleted in https://github.com/go-mysql-org/go-mysql/pull/959

I think you need `utils.ByteSliceToString`